nim/scorper/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  github: bung87/scorper
+  version: 1.0.7
```

could you please use `X.Y` format ? (this decrease maintenance tasks)

```suggestion
  version: 1.0
```